setup.py

```diff
@@ -111,8 +111,10 @@ def run(self):
             self.do_egg_install()
 
         if not os.path.exists(docs_compiled_dest):
-            print('warning: Sphinx "htmlhelp" documentation was '
-                  'NOT bundled!', file=sys.stderr)
+            print('\nwarning: Sphinx "htmlhelp" documentation was NOT bundled!\n'
+                  '         Execute the following then reinstall:\n\n'
```

Comment:
Let me understand this: 1) users have to run install to see this message; 2) users need to run `build_sphinx`; 3) users need to run install AGAIN (=re-install)??? Isn't this too much?